src/ShellClients/PanelClone.vala

```diff
@@ -79,54 +64,22 @@ public class Gala.PanelClone : Object {
         });
     }
 
-    private void update_visible () {
-        actor.visible = !panel_hidden;
-
-        if (actor.visible && !wm.get_display ().get_monitor_in_fullscreen (panel.window.get_monitor ())) {
-            // The actor has just been revealed, make sure it's at the top
-            // https://github.com/elementary/gala/issues/2080
-            actor.get_parent ().set_child_above_sibling (actor, null);
-        }
-    }
-
-    private void update_clone_position () {
-        clone.set_position (calculate_clone_x (panel_hidden), calculate_clone_y (panel_hidden));
-    }
-
-    private float calculate_clone_x (bool hidden) {
-        switch (panel.anchor) {
-            case TOP:
-            case BOTTOM:
-                return actor.x;
-            default:
-                return 0;
-        }
-    }
-
-    private float calculate_clone_y (bool hidden) {
+    private float calculate_y (bool hidden) {
```

Comment:
I think 'calculate_y' is ambiguous in this case because it calculates translation not coordinates, maybe 'calculate_translation_y' would be better?

Reply:
Updated it :)